spec/system/visit_sign_in_spec.rb

```diff
@@ -111,4 +111,25 @@
     expect(page.get_rack_session.has_key?(:challenge_parameters)).to eql false
   end
 
+  scenario 'User get prompted to setup MFA on first sign in' do
```

Comment:
one last comment... we should also have a failure test, i.e. testing the error message etc. The one which is being deleted.